lighthouse-core/scripts/pptr-run-devtools.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * USAGE:
+ * Make sure CHROME_PATH is set to a modern version of Chrome.
+ * This script won't work on older versions that use the "Audits" panel.
+ *
+ * URL list file: yarn run-devtools < path/to/urls.txt
+ * Single URL: yarn run-devtools "https://example.com"
+ */
+
+const puppeteer = require('puppeteer');
+const fs = require('fs');
+const readline = require('readline');
+const yargs = require('yargs/yargs');
+
+/** @typedef {{result?: {value?: string, objectId?: number}, exceptionDetails?: object}} RuntimeEvaluateResponse */
+
+const argv = yargs(process.argv.slice(2))
+  .usage('$0 [url]')
+  .help('h').alias('h', 'help')
+  .option('_', {type: 'string'})
+  .option('o', {
+    type: 'string',
+    default: 'latest-run/devtools-lhrs',
+  }).alias('o', 'output-dir')
```

Comment:
not a big deal, but I find it to be more readable for non-authors when the option is the typical JS camelCase version and the alias is the letter. You still get the same support for `-o`, `--output-dir` and `--outputDir` but it's much clearer in the JS when you use `argv.outputDir` and it matches the explicit option :)

Reply:
I'm a little conflicted here.

I think you're right that the readability is better, but I would also prefer to keep `--custom-devtools-frontend` in kebab case because it matches the corresponding flag used on the chrome instance. It's probably better to keep the case for these two flags the same.

I think I will leave this as as for now.

Reply:
that's fine. fwiw I'd still take `argv['custom-devtools-frontend']` over `argv.d` 😉